### PR TITLE
Added/Updated tests

### PR DESCRIPTION
**Pull request by shreyanshdwivedi**
Related to #3 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [ ] Added Surge preview link

#### Changes proposed in this pull request:

-I have added/updated tests in order to increase coverage.

### CODE REVIEW

**Comment by shreyanshdwivedi** [comment]
@praveenojha33 @AakashMallik @prabhupant please review.

File: src/app/about/about.component.spec.ts [hunk 1]
```diff
@@ -46,6 +46,12 @@ describe('Component: About', () => {
     expect(footerNavbar).toBeTruthy();
   });
 
+  it('should have a footer element', () => {
+    let compiled = fixture.debugElement.nativeElement;
```
Comment (simsausaurabh) [inline src/app/about/about.component.spec.ts]: Use ```const```

File: src/app/advancedsearch/advancedsearch.component.spec.ts [hunk 2]
```diff
@@ -40,7 +40,16 @@ describe('AdvancedsearchComponent', () => {
...
+  it('should create an advanced search instance', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should have advance search modal', () => {
+    let compiled = fixture.debugElement.nativeElement;
+
+    let button = compiled.querySelector('button');
+    button.click();
+    let modalBody: HTMLInputElement = compiled.querySelector('div.modal-body');
```
Comment (simsausaurabh) [inline src/app/advancedsearch/advancedsearch.component.spec.ts]: Use ```const```

File: src/app/advancedsearch/advancedsearch.component.spec.ts [hunk 3]
```diff
@@ -40,7 +40,16 @@ describe('AdvancedsearchComponent', () => {
     fixture.detectChanges();
   });
 
-  it('should create an instance', () => {
+  it('should create an advanced search instance', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should have advance search modal', () => {
+    let compiled = fixture.debugElement.nativeElement;
```
Comment (simsausaurabh) [inline src/app/advancedsearch/advancedsearch.component.spec.ts]: Use ```const```

File: src/app/advancedsearch/advancedsearch.component.spec.ts [hunk 4]
```diff
@@ -40,7 +40,16 @@ describe('AdvancedsearchComponent', () => {
...
+
+    let button = compiled.querySelector('button');
```
Comment (simsausaurabh) [inline src/app/advancedsearch/advancedsearch.component.spec.ts]: Use ```const```

File: src/app/contact/contact.component.spec.ts [hunk 5]
```diff
@@ -56,6 +56,12 @@ describe('Component: Contact', () => {
     expect(footerNavbar).toBeTruthy();
   });
 
+  it('should have a footer element', () => {
+    let compiled = fixture.debugElement.nativeElement;
```
Comment (simsausaurabh) [inline src/app/contact/contact.component.spec.ts]: Use ```const```

File: src/app/about/about.component.spec.ts (hunk 1 above)
Comment (shreyanshdwivedi) [reply src/app/about/about.component.spec.ts]: @simsausaurabh should I change all `let` instances to `const` instance or just the instances I added?

**Comment by shreyanshdwivedi** [comment]
@praveenojha33 @simsausaurabh I've made required changes and squashed the commits. Please review

File: src/app/about/about.component.spec.ts (hunk 1 above)
Comment (simsausaurabh) [reply src/app/about/about.component.spec.ts]: Please change all instances in this file and check if the tests pass correctly.

**Comment by simsausaurabh** [comment]
I am not sure about it, please see if this is the valid reason: https://github.com/nrwl/nx/issues/582#issuecomment-413984596

File: src/app/about/about.component.spec.ts (hunk 1 above)
Comment (shreyanshdwivedi) [reply src/app/about/about.component.spec.ts]: @simsausaurabh I changed all the `let` instances to `const` instance except on 8th and 9th line. And it is passing `ng test` successfully.

**Comment by praveenojha33** [comment]
Please open an issue so that everyone can provide their input on it.

**Comment by shreyanshdwivedi** [comment]
Opened issue #1296 for coverage report problem.

File: src/app/about/about.component.spec.ts (hunk 1 above)
Comment (shreyanshdwivedi) [reply src/app/about/about.component.spec.ts]: @simsausaurabh Please update me. How should I proceed?

Comment (shreyanshdwivedi) [reply src/app/about/about.component.spec.ts]: @simsausaurabh @praveenojha33  Please update. I need to proceed in this.

**Comment by shreyanshdwivedi** [comment]
@praveenojha33 @simsausaurabh I've changes all `let` instances to `const` and it is passing all the tests on my local. I think Travis is failing bcoz it's currently down in our repo.

**Comment by simsausaurabh** [comment]
Please squash the commit into one. Then it will be good to be merged.

**Pull request merged**